packages/nuxt/src/core/nitro.ts

```diff
@@ -369,8 +402,30 @@ export async function initNitro (nuxt: Nuxt & { _nitro?: Nitro }) {
       await prepare(nitro)
       await copyPublicAssets(nitro)
       await nuxt.callHook('nitro:build:public-assets', nitro)
+
+      // Add pages prerendered but not covered by route rules
+      const prerenderedRoutes = new Set<string>()
+      const routeRulesMatcher = toRouteMatcher(
+        createRadixRouter({ routes: nitro.options.routeRules })
+      )
+      const payloadSuffix = nuxt.options.experimental.renderJsonPayloads ? '/_payload.json' : '/_payload.js'
+      nitro.hooks.hook('prerender:route', (route) => {
+        if (!route.error && route.route.endsWith(payloadSuffix)) {
+          const url = route.route.slice(0, -payloadSuffix.length) || '/'
+          const rules = defu({}, ...routeRulesMatcher.matchAll(url).reverse()) as Record<string, any>
+          if (!rules.prerender) {
+            prerenderedRoutes.add(url)
+          }
+        }
+      })
       await prerender(nitro)
 
+      for (const file of ['latest.json', `meta.${buildId}.json`]) {
+        const manifestFile = join(nitro.options.output.publicDir, '_builds', file)
```

Comment:
Initially I was thinking directly `_nuxt/builds` as it's a build asset - updated as there was an issue when prerendering - let me check again....

Reply:
let's leave it as `_builds` for now but iterate (happy to rename when nitro feature comes out). Other options could be `_meta` or to keep it as `_nuxt/builds`

Reply:
`_nuxt/builds` can be tricky because `_nuxt` itself is part of a single build.

`_builds` could also be tricky to align next minor versions of nitro because it might override nuxt one. Why not using `_nuxtBuilds` for now?

Reply:
aesthetic reasons only - I don't think it looks or sounds nice. I want it to sound generic, without `nuxt` included in the name

Reply:
🤷 I think for the same reason we use `_nuxt` as the default build dir (however customizable). It is really important otherwise my hands will be tied to even try introducing the feature in nitro!! (same for users to leverage a directory called `_build` they won't be able to)

Reply:
previously you said:
> however i think for nitro we should also support custom name so it is nuxt compatible

so I thought it would be fine to use `_builds` for nuxt

Reply:
By default, it will generate it when landed. PR thinking more on this topic here, i think even for nuxt is unsafe to use a generic name like `_builds`. Users might need a public directory like this and it will conflict without vendor prefix.

Reply:
What generic name will you use in Nitro? I can avoid it then.

Reply:
I guess what I am trying to say is, for nitro, it should also be `_nitro/builds` or something with vendor prefix but nuxt probably shouldn't also use `_builds`. it was something unforeseen in our early discussions. (issue is not nitro anymore but also for nuxt to be non conflicting)

Reply:
It can still be a required path for users. Same as `/.well-known/` which is however starting with dot, it is a valid path and might be needed.

Reply:
As with nitro, I think we can make it configurable if users need an escape hatch. 👍